activesupport/lib/active_support/array_inquirer.rb

```diff
@@ -9,8 +9,10 @@ module ActiveSupport
   #   variants.desktop?  # => false
   class ArrayInquirer < Array
     # Passes each element of +candidates+ collection to ArrayInquirer collection.
-    # The method returns true if at least one element is the same. If +candidates+
-    # collection is not given, method returns true.
+    # The method returns true if any element from the +candidates+ collection
+    # matches stringified or symbolized form of any element in the ArrayInquirer collection.
```

Comment:
@vipulnsward when using `the element` it could refer to a specific element. But `any element` actually refers to just any element in the collection. Am I missing something here ?